src/csharp/Microsoft.Spark/ML/Feature/StopWordsRemover.cs

```diff
@@ -0,0 +1,150 @@
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        public StopWordsRemover SetLocale(string value) =>
+            WrapAsStopWordsRemover(_jvmObject.Invoke("setLocale", value));
+
+        /// <summary>
+        /// Gets locale for <see cref="StopWordsRemover"/> transform
+        /// </summary>
+        /// <returns>string, the local</returns>
+        public string GetLocale() => (string)(_jvmObject.Invoke("getLocale"));
+
+        /// <summary>
+        /// Sets case sensitivity for <see cref="StopWordsRemover"/> transform
+        /// </summary>
+        /// <param name="value">case sensitivity to be used for transform</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        public StopWordsRemover SetCaseSensitive(bool value) =>
+            WrapAsStopWordsRemover(_jvmObject.Invoke("setCaseSensitive", value));
+
+        /// <summary>
+        /// Gets case sensitivity for <see cref="StopWordsRemover"/> transform
+        /// </summary>
+        /// <returns>string, Is case sensitive?</returns>
+        public bool GetCaseSensitive() => (bool)(_jvmObject.Invoke("getCaseSensitive"));
+
+        /// <summary>
+        /// Sets custom stop words for <see cref="StopWordsRemover"/> transform
+        /// </summary>
+        /// <param name="values">Custom stop words</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        public StopWordsRemover SetStopWords(IEnumerable<string> values) =>
+            WrapAsStopWordsRemover(_jvmObject.Invoke("setStopWords", values));
+
+        /// <summary>
+        /// Gets custom stop words for <see cref="StopWordsRemover"/> transform
+        /// </summary>
+        /// <returns>string, Custom stop words</returns>
+        public string[] GetStopWords() => (string[])(_jvmObject.Invoke("getStopWords"));
+
+        /// <summary>
+        /// Validate and get the transform schema for <see cref="StopWordsRemover"/> transform
+        /// </summary>
+        /// <returns>string, Output schema</returns>
+        public StructType TransformSchema(StructType value) =>
+            new StructType(
+                (JvmObjectReference)_jvmObject.Invoke("transformSchema",
+                    DataType.FromJson(_jvmObject.Jvm, value.Json)));
+
+        /// <summary>
+        /// Loads the <see cref="StopWordsRemover"/> that was previously saved using Save
+        /// </summary>
+        /// <param name="path">The path the previous <see cref="StopWordsRemover"/> was saved to</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object, loaded from path</returns>
+        public static StopWordsRemover Load(string path)
+        {
+            return WrapAsStopWordsRemover(
+                SparkEnvironment.JvmBridge.CallStaticJavaMethod(
+                    s_stopWordsRemoverClassName, "load", path));
+        }
+
+        JvmObjectReference IJvmObjectReferenceProvider.Reference => _jvmObject;
```

Comment:
This belongs at the top after the constructors, before the public methods (see https://github.com/dotnet/spark/blob/21edf80bb0f3bc2c49a397abc97e063168fc9367/src/csharp/Microsoft.Spark/ML/Feature/FeatureHasher.cs#L31)